CHANGELOG.md

```diff
@@ -19,8 +33,14 @@ All notable changes to this project will be documented in this file. The format
 
 - **Documentation:**
   - Raise awareness in the troubleshooting page for a common misconfiguration when deviating from our advice by using a bare domain ([#3680](https://github.com/docker-mailserver/docker-mailserver/pull/3680))
+  - updated note about Rspamd on ARM64 lacking behind in version
```

Comment:
https://github.com/docker-mailserver/docker-mailserver/issues/3374#issuecomment-1844793514

> By now there are ARM64 builds in the official rspamd debian repository. So it may be installed from this source instead of debian backports.

Recently merged PR that removed the docs addition, these changes were for v13.1: https://github.com/docker-mailserver/docker-mailserver/pull/3686

The `package.sh` also reflects that update, so this changelog entry (with no PR) might as well be dropped?

```suggestion
```